app/src/main/java/org/jellyfin/androidtv/util/profile/MediaCodecCapabilitiesTest.kt

```diff
@@ -3,11 +3,23 @@ package org.jellyfin.androidtv.util.profile
 import android.media.MediaCodecInfo.CodecProfileLevel
 import android.media.MediaCodecList
 import android.media.MediaFormat
+import android.os.Build
+import androidx.annotation.RequiresApi
 import timber.log.Timber
 
 class MediaCodecCapabilitiesTest {
 	private val mediaCodecList by lazy { MediaCodecList(MediaCodecList.REGULAR_CODECS) }
 
+	@RequiresApi(Build.VERSION_CODES.Q)
+	fun supportsAV1(): Boolean = hasCodecForMime(MediaFormat.MIMETYPE_VIDEO_AV1)
+
+	@RequiresApi(Build.VERSION_CODES.Q)
+	fun supportsAV1Main10(): Boolean = hasDecoder(
+		MediaFormat.MIMETYPE_VIDEO_AV1,
+		CodecProfileLevel.AV1ProfileMain10,
+		CodecProfileLevel.AV1Level5
+	)
```

Comment:
I'd prefer it if those functions checked the API version instead of the calling side. Both can just return `false` for anything below Android Q